static/locales/en-US.yaml

```diff
@@ -331,6 +401,9 @@ Settings:
     Are you sure you want to remove all subscriptions and profiles?  This cannot be undone.: Are
       you sure you want to remove all subscriptions and profiles?  This cannot be
       undone.
+    Remove All Playlists: Remove All Playlists
+    All playlists have been removed: All playlists have been removed
+    Are you sure you want to remove all your playlists?: Are you sure you want to remove all your playlists?
```

Comment:
```suggestion
    Are you sure you want to remove all of your playlists?: Are you sure you want to remove all of your playlists?
```

Reply:
Any reason to use `all of`?
If it feels inconsistent with `Are you sure you want to remove all subscriptions and profiles?` I can remove `your` (not sure why I add `your`)

Reply:
I personally think it sounds more formal, but after checking online, [English is weird and both are completely fine](https://english.stackexchange.com/questions/80412/is-of-necessary-in-all-of). Fine to close this one if you want.